# Generated by Django 3.2.3 on 2021-07-12 13:16

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0006_rename_langauge_customuser_language'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='customuser',
            name='is_subscribed_newsletters',
        ),
        migrations.RemoveField(
            model_name='customuser',
            name='language',
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
